packages/rocketchat-theme/client/imports/base.css

```diff
@@ -4745,13 +4745,13 @@ a + br.only-after-a {
 }
 
 .embedded-view {
-	& .flex-tab-bar {
-		display: none;
-	}
-
 	& .messages-container {
 		border-width: 0;
 
+		& .flex-tab-container {
```

Comment:
:thinking: I don't get it.. there is no space *after* the dot and it's exactly the same as all other selectors =)

also you can see it working [on heroku](https://rocket-chat-pr-7486.herokuapp.com/channel/general?layout=embedded) and the selector being used by google chrome:
![image](https://user-images.githubusercontent.com/8591547/28210924-4f12c1b6-6870-11e7-99a2-73cbd3cb97b6.png)